cmd/run.php

```diff
@@ -41,3 +42,24 @@
 };
 
 $climate->out($result);
+
+$climateBrewery = new CLImate();
+$nameBrewery = $climateBrewery->input("Please provide brewery or city name:")->prompt();
+$scrapperBrewery = new BreweryScrapper($nameBrewery, new Client());
+echo PHP_EOL;
+$scrapperBrewery->getShowLabel($nameBrewery);
+while (true) {
+    $optionBrewery = $climateBrewery->radio(PHP_EOL . "Choose data to fetch:", [
+        BreweryScrapper::LIST => "showing a list of providers",
+        BreweryScrapper::CLEAR => "clearing cache",
+        BreweryScrapper::EXIT => "Exit",
+    ])->prompt();
+
+    $resultBrewery = match ($optionBrewery) {
+        BreweryScrapper::LIST => $scrapperBrewery->getList(),
+        BreweryScrapper::CLEAR => $scrapperBrewery->getClear(),
+        BreweryScrapper::EXIT => $scrapperBrewery->getExit(),
+    default => "unknown option",
```

Comment:
You should return this message as an error.
`$climate->error('something happen');`
or 
`$climate->to('error')->out('something happen');`